<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(225, 213, 231);
}
</string>
  </property>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>YYYY - MM - DD DDD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string> Home Chill Main</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_chillguy">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>30</y>
     <width>131</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Camera">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
font-size: 20px;
padding: 10px;
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Camera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-radius: 10px;  /* 모서리 둥글게 */
    border: 2px solid lightgray; /* 테두리 추가 (기본 버튼 색 유지) */
    background-color: palette(button); /* 기본 버튼 색 유지 */
    color: palette(button-text); /* 기본 글자 색 유지 */

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>480</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid gray;  /* 테두리 */
border-radius: 5px;  /* 모서리 둥글게 */
</string>
   </property>
   <property name="text">
    <string>Gas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>240</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid gray;  /* 테두리 */
border-radius: 5px;  /* 모서리 둥글게 */
</string>
   </property>
   <property name="text">
    <string>Garage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>400</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid gray;  /* 테두리 */
border-radius: 5px;  /* 모서리 둥글게 */
</string>
   </property>
   <property name="text">
    <string>Window</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>160</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid gray;  /* 테두리 */
border-radius: 5px;  /* 모서리 둥글게 */
</string>
   </property>
   <property name="text">
    <string>LED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider_garage">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_window">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>320</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid gray;  /* 테두리 */
border-radius: 5px;  /* 모서리 둥글게 */
</string>
   </property>
   <property name="text">
    <string>Door</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider_door">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_led">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: 	#72CC82; /* 연한 하늘색 */
	border: 1px solid gray;  /* 테두리 */
    border-radius:  19px;  /* 모서리를 둥글게 (값을 조절 가능) */</string>
   </property>
   <property name="text">
    <string>Safe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 0, 0); /* 연한 하늘색 */
	border: 1px solid gray;  /* 테두리 */
    border-radius:  19px;  /* 모서리를 둥글게 (값을 조절 가능) */</string>
   </property>
   <property name="text">
    <string>Danger</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: 	#FF8400; /* 연한 하늘색 */
	border: 1px solid gray;  /* 테두리 */
    border-radius:  19px;  /* 모서리를 둥글게 (값을 조절 가능) */</string>
   </property>
   <property name="text">
    <string>Caution</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_move_to_log">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>59</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color: rgb(225, 213, 231);
    border-radius:  19px;  /* 모서리를 둥글게 (값을 조절 가능) */
}
</string>
   </property>
   <property name="text">
    <string>LOG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_move_to_users">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color: rgb(225, 213, 231);
    border-radius:  20px;  /* 모서리를 둥글게 (값을 조절 가능) */
	border: 1px solid rgb(150, 115 ,160);  /* 테두리 */
}
</string>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_name_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <zorder>label_chillguy</zorder>
  <zorder>label_date</zorder>
  <zorder>label_12</zorder>
  <zorder>label_Camera</zorder>
  <zorder>btn_Camera</zorder>
  <zorder>label_8</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>slider_garage</zorder>
  <zorder>slider_window</zorder>
  <zorder>label_5</zorder>
  <zorder>slider_door</zorder>
  <zorder>slider_led</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>btn_move_to_log</zorder>
  <zorder>btn_move_to_users</zorder>
  <zorder>label_user_name</zorder>
  <zorder>label_user_name_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
